File: beginner/javascript/2006.js
let input = require('fs').readFileSync('stdin', 'utf-8').split('\n');

var cont = 0;
    
n = parseInt(input[0]);
var valores = input[1].split(" ");

for(i = 0; i < 5; i++){
    if(parseInt(valores[i]) == n)
        cont++;
}

console.log(cont);

File: beginner/c/2006.c
#include <stdio.h>
 
int main() {
 
    int n, i, v, cont = 0;
    
    scanf("%d", &n);
    
    for(i = 0; i < 5; i++){
        scanf("%d", &v);
        if(v == n)
            cont++;
    }
    
    printf("%d\n", cont);
 
    return 0;
}